test/react_client/bridge_test.dart

```diff
@@ -0,0 +1,71 @@
+@TestOn('browser')
+library react.bridge_test;
+
+import 'package:react/react.dart' as react;
+import 'package:react/react_client.dart';
+import 'package:react/react_client/bridge.dart';
+import 'package:test/test.dart';
+
+import '../util.dart';
+
+main() {
+  setClientConfiguration();
+
+  // Test behavior that isn't functionally tested via component lifecycle tests.
+  group('Component2Bridge', () {
+    group('.forComponent returns the bridge used by that mounted component, which', () {
+      test('by default is a const instance of Component2BridgeImpl', () {
+        final instance = getDartComponent<_Foo>(render(Foo({})));
+        expect(Component2Bridge.forComponent(instance), same(const Component2BridgeImpl()));
+      });
+
+      test('can be a custom bridge specified via registerComponent2', () {
+        customBridgeCalls.clear();
```

Comment:
Should this be put into a `setUp`?

Reply:
Since it's only used in this one test, I didn't think it was necessary.

Reply:
I'll improve this, though

Reply:
Added https://github.com/cleandart/react-dart/pull/208/commits/57bfbfd90d70e6f302fbc91a7c1c238d50d1c156 which I think is cleaner than what I had before